macapype/pipelines/full_pipelines.py

```diff
@@ -500,116 +504,13 @@ def create_full_segment_pnh_subpipes(
 
     return seg_pipe
 
+# ################# ANTS based segmentation for adrien baboons (T1 without T2)
 
-def create_full_segment_pnh_noT2atlasbrex_subpipes(
```

Comment:
@kepkeeloh Actually before merging: I removed this part create_full_segment_pnh_noT2atlasbrex_subpipes that was used nowhere in your code; can you confirm this is useless and can be removed ?